dfinity/verifiable-credentials-sdk
# Return decoded info

# Motivation

Library should be user-friendly and also expose the decoded JWT information (not verified).

# Changes

* Install `jose` dependency to decode the JWT.
* Create new types for the argument passed to the `onSuccess` function.
* New helper function `decodeCredentials`.

# Tests

* Add mocks file with a valid JWT token to be decoded in the tests along with the expected decoded value.
* Change the expected value of `onSuccess` in the tests.
* Add `unreachableFn` helper instead of checking that the callback wasn't called.
* Add a new test when decoding fails.
